pkg/azure/actuator.go

```diff
@@ -58,28 +62,58 @@ func (a *Actuator) IsValidMode() error {
 	return errors.New("invalid mode")
 }
 
+func isAzureCredentials(providerSpec *runtime.RawExtension) (bool, error) {
+	codec, err := minterv1.NewCodec()
+	if err != nil {
+		return false, err
+	}
```

Comment:
Decode as unknown to check for Kind.

Reply:
Way better than what I suggested, nice.

Reply:
I think it could be pushed up to the predicates function, or even above that when we register the actuators. but that requires more refactoring, so this should do for now.
Do we let the AWS implementation log errors on azure creds for now?

Reply:
Are azure cred requests landing in the release image yet? (these come from each component's repo that needs creds for AWS)

Reply:
in Ingress Operator PR :
https://github.com/openshift/cluster-ingress-operator/pull/231/files